pkg/airgapped/plugin_bundle_download.go

```diff
@@ -173,6 +176,21 @@ func (o *DownloadPluginBundleOptions) getSelectedPluginInfo() ([]*plugininventor
 		}
 	}
 
+	for _, pluginID := range o.Plugins {
+		pluginName, pluginTarget, pluginVersion := utils.ParsePluginID(pluginID)
+		pluginEntries, err := pi.GetPlugins(&plugininventory.PluginInventoryFilter{
+			Name:          pluginName,
+			Target:        configtypes.Target(pluginTarget),
+			Version:       pluginVersion,
+			IncludeHidden: true,
+		}) // Include the hidden plugins during plugin migration
+
+		if err != nil {
+			return nil, nil, errors.Wrap(err, "unable to read plugins from database")
+		}
```

Comment:
I think we should check if 0 plugins were returned which would mean the value of `--plugin` is wrong.
For example, if the user typed `--plugin cluster:kubernetes` it won't download the plugin but will not print an error either.

Reply:
Could we add a unit test to check that the command fails if the plugin specified is not found?